### PR TITLE
html forms

### PR DESCRIPTION
**Pull request by jeesupark96**
https://user-images.githubusercontent.com/90468158/144109467-11156261-064c-4f15-859c-12b8ae9a7ad9.mp4

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
```
Comment (BrettAlbright) [inline html-forms/index.html]: There should only be one form on the page for all of the inputs. so make sure to get rid of the extra forms and just have one

Comment (BrettAlbright) [inline html-forms/index.html]: the action attribute is not needed on this particular form, but we will discuss what it does in Q and A

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      <label>
```
Comment (BrettAlbright) [inline html-forms/index.html]: need a for attribute here which matching the id on the input below

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        <div>Email</div>
+        <input type="email" name="Email" id="email" />
+      </label>
+    </form>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
+      <div>
+        <label for="why is coding cool">Why is coding cool</label>
+      </div>
+      <select name="Why is coding cool" id="why is coding cool">
+        <option value="Select and option">Select an Option</option>
```
Comment (BrettAlbright) [inline html-forms/index.html]: when the option is used for a default message, you do not need to include a value for it

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+    </form>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
+      <div>
+        <label for="Favorite Language (choose one)"
+          >Favorite Language (choose one)</label
+        >
+      </div>
+      <input type="radio" name="Language" value="HTML" />
```
Comment (BrettAlbright) [inline html-forms/index.html]: you will need to add an id attribute to the radio inputs which match the for attribute on the labels

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        name="More Information"
+        value="Sign me Up"
+        id="Sign Up" />
+      <label for="Sign Me Up">Sign Me Up!</label>
+    </form>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <textarea name="Comments" cols="20" rows="7" id="Comments"> </textarea>
```
Comment (BrettAlbright) [inline html-forms/index.html:51]: watch out for the capital `c` in comments! it doesn't match the `c` in the label.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Very nice work on this! There are some updates to complete, let me know if you have any questions!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**